app/services/article_api_index_service.rb

```diff
@@ -81,6 +84,15 @@ def state_articles(state)
     end
   end
 
+  def collection_articles(collection_id)
+    Article.published.
+      where(collection_id: collection_id).
+      includes(:user, :organization).
+      order("collection_id").
```

Comment:
I don't think one is needed, since they are filtered by collection id, they should probably be ordered by published time in ascending order, as in the first article should be the oldest of the series

Reply:
Oops. I have forgotten to pass the collection_id in the test .... I have also modified the order in the service.